api/test_api.py

```diff
@@ -1,2 +1,27 @@
-# pylint: ignore-pylint
-from api import *
+# pylint: skip-file
+from dotenv import load_dotenv
+import pytest
+from api import app
+from unittest.mock import patch
+
+load_dotenv()
+
+
+@pytest.fixture
+def client():
+    app.config["TESTING"] = True
+    with app.test_client() as client:
+        yield client
+
+
+def test_endpoint_index(client):
+    response = client.get("/")
+    assert response.status_code == 200
+    assert "message" in response.json
+
+
+def test_endpoint_get_earthquakes(client):
+    response = client.get("/earthquakes")
+    print(response.json)
```

Comment:
this will be unnecessary later i think?

Reply:
Why do you think so?

Reply:
The print statement I mean. Could use logging instead?